lib/matrix.gd

```diff
@@ -71,13 +71,22 @@ DeclareProperty( "IsGeneralizedCartanMatrix", IsMatrix );
 ##  <Oper Name="IsDiagonalMat" Arg='mat'/>
 ##
 ##  <Description>
-##  returns true if mat has only zero entries off the main diagonal, false
-##  otherwise.
+##  return <K>true</K> if the matrix <A>mat</A> has only zero entries
+##  off the main diagonal, and <K>false</K> otherwise.
```

Comment:
Not an issue of this PR, as the text is old, but: I always found this text confusing and usually have to read it at least twice. And in light of the possibility of unbound entries in a classical matrix (list of lists), one could even misinterpret this as saying: The diagonal entries are all unbound, and any bound entry outside the diagonal must be zero. Which of course is nonsense, nobody would read it that way, but still, it irks me.

How about this instead: "returns true if all off-diagonal entries of mat are zero, and false otherwise." ?